service-account/bound-sa-token.md

```diff
@@ -1,122 +0,0 @@
-%title: Bound Service Account Token
-%author: Krzysztof Ostrowski
-%date: 10.03.2023
-
--> # Bound Service Account Token <-
-
---------------------------------------------------
--> # Table of Contents <-
-==============
-
--> # What is a Service Account Token  <-
--> # Service Account Token as a YAML <-
```

Comment:
there's no reason to present SA tokens as yamls

Reply:
Yes, this is true. I meant Service Account.